CodyJones4235/PoopMap
# halfway there

#### Changes Made
1. Modified files to include map, map markers, and user posts and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cwj-getting-started
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Posts & map should display!
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
